sass/_theme_rst.sass

```diff
@@ -17,7 +17,7 @@
   // Sphinx by default applies HxW style attributes to images. This fixes that oversite.
   img
     max-width: 100%
-    height: auto !important
+    height: auto 
```

Comment:
Not sure here but maybe a better solution would be to set `width: auto !important`